gtfs/gtfs_utils/gtfs_utils/configuration.py

```diff
@@ -77,15 +79,15 @@ class Configuration:
     console_verbosity: str = 'ERROR'
 
 
-def dict_to_dataclass(data_dict: Dict, data_class: type) -> Dict:
+def dict_to_dataclass(data_dict: Dict, data_class: type) -> Configuration:
     """
     Converts the dict to a dataclass instance of the given type.
     """
     data_class_instance = data_class()
 
     for field in fields(data_class):
         if field.name not in data_dict:
-            value = field.default
+            continue  # The value is already set by default value of the class
```

Comment:
Are you sure that's OK? did you test it?

Reply:
Yes, The default values are set when we initialize the instance (`data_class_instance = data_class()`).
The only difference now is that it skips the type check when the default value is used, I thought it's redundant to validate the type for default values but I can change that.

Reply:
I agree, just making sure it's tested :)

Reply:
Actually, if it's OK this way, you can even change it to a positive condition:
Instead of using 
```python
if field.name not in data_dict:
	continue
else:
	...
```

you can simply change it to:
```python
if field.name in data_dict:
	...
```